core/arch/arm/kernel/link.mk

```diff
@@ -28,7 +28,8 @@ link-script-cppflags := -DASM=1 \
 entries-unpaged += thread_init_vbar
 entries-unpaged += sm_init
 entries-unpaged += core_init_mmu_regs
-entries-unpaged += stack_tmp_top
+entries-unpaged += stack_tmp
```

Comment:
`KEEP_PAGER(stack_tmp)` and `KEEP_PAGER(stack_tmp_stride)` in `thread.c` should make this redundant.